dotCMS/src/main/java/com/dotmarketing/portlets/variant/business/VariantAPI.java

```diff
@@ -0,0 +1,58 @@
+package com.dotmarketing.portlets.variant.business;
+
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.variant.model.Variant;
+import java.util.Optional;
+
+public interface VariantAPI {
```

Comment:
All these classes of API and Factory are in the wrong package. They need to be under "com.dotcms.variants" business/model

Reply:
done https://github.com/dotCMS/core/pull/22839/commits/8aaaf9e8f7d06c71edfd820737a9fcabc12d94ed#diff-a12949e743698a4a040ef3bdb19254e41063e5c10505f5ece12b8a30fe761ef5R1

Reply:
approved